sunpy/map/sources/rhessi.py

```diff
@@ -45,14 +45,15 @@ def __init__(self, data, header, **kwargs):
         # than one image (schriste)
         # Fix some broken/misapplied keywords
         if self.meta['ctype1'] == 'arcsec':
-            self.meta['cunit1'] = 'arcsec'
-            self.meta['ctype1'] = 'HPLN-TAN'
+            self.meta['cunit1'] = self.meta.get('cunit1', 'arcsec')
+            self.meta['ctype1'] = self.meta.get('ctype1', 'HPLN-TAN')
         if self.meta['ctype2'] == 'arcsec':
-            self.meta['cunit2'] = 'arcsec'
-            self.meta['ctype2'] = 'HPLT-TAN'
+            self.meta['cunit2'] = self.meta.get('cunit2', 'arcsec')
+            self.meta['ctype2'] = self.meta.get('ctype2', 'HPLT-TAN')
```

Comment:
This metadata overwriting appears to be necessary with the RHESSI headers.  `CTYPEi` mistakenly contains `CUNITi` information.

Reply:
If you want to be safe, you can first check whether `CTYPEi` contains "arcsec", and if so, replace with "HPLT-TAN".

Reply:
ah yes. Is that true for all RHESSI image headers?

Reply:
I haven't checked